posts/2020/02/my-unexpected-dive-into-open-source-python.md

```diff
@@ -55,7 +55,7 @@ I look forward to contributing to the open-source community!
 
 ## Special Thanks to:
 - PyData's Diversity Scholars program and its NYC volunteer organizers Hanhan and Vinay.
-- My new friends that welcomed me at PyData NYC- W4rner, Bhargav, Agatha and Lev.
+- My new friends that welcomed me at PyData NYC- W4rner, Bhargav, Agata and Lev at their [Natural Language Role-Playing workshop](https://agileeducation.co/nlrp), hosted by [agileEducation](https://agileeducation.co/).
```

Comment:
while we're at it in this sentence: 
- should `W4rner` be `Werner`? EDIT: I see at https://agileeducation.co/. Should probably either name him Laurence, or make his handle a link
- should there be a space in `NYC-`

Reply:
It is a little oddly spelled, but it is indeed W4rner, his online handle he requested.

For the space in 'NYC- ', my google searches say its a stylistic choice if there are spaces before and after the dash. I decided to put in a space for clarity.

Reply:
Okay, thanks for checking Mars.